.circleci/config.yml

```diff
@@ -49,11 +70,6 @@ jobs:
               sudo tar -zxvf geckodriver-v0.19.0-linux64.tar.gz -C /opt
```

Comment:
This might not be needed, the circle browsers image [installs](https://github.com/circleci/circleci-images/blob/0d8b6758dc1214046a3f5e7f14297b1a8572b479/shared/images/Dockerfile-browsers.template#L32) the `geckodriver`